quic/s2n-quic-core/src/recovery/rtt_estimator.rs

```diff
@@ -15,6 +15,10 @@ use core::{
 //# starting with a PTO of 1 second, as recommended for TCP's initial
 //# RTO; see Section 2 of [RFC6298].
 pub const DEFAULT_INITIAL_RTT: Duration = Duration::from_millis(333);
+
+/// The lowest initial RTT value that the RTT Estimator is capable of tracking
+pub const MIN_INITIAL_RTT: Duration = Duration::from_micros(1);
```

Comment:
You could just call it `MIN_RTT` and also use it in `update_rtt` and the `zero_rtt_sample` test

Reply:
can do